pro_wes/ga4gh/wes/workflow_runs.py

```diff
@@ -437,18 +455,20 @@ def _process_attachments(self, work_dir: Path) -> List[Attachment]:
         attachments = []
         files = request.files.getlist("workflow_attachment")
         for file in files:
-            attachments.append(
-                Attachment(
-                    filename=file.filename,
-                    object=file.stream,
-                    path=str(
-                        work_dir
+            if file is not None:
+                if file.filename is None:
+                    raise ValueError("File does not have a filename.")
```

Comment:
I would use assert here to avoid the branching. I don't know exactly under what conditions `file.filename` could be `None`, but if we will ever run into this issue, then an `AssertionError` will be just as informative as the `ValueError` you propose. And then we can still look at the cause of the issue in detail and find a solution for that (e.g., create a random file name or whatever).